pkg/datapath/linux/config/config.go

```diff
@@ -536,6 +537,32 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_CUSTOM_CALLS"] = "1"
 	}
 
+	if option.Config.EnableVTEP {
+		cDefinesMap["ENABLE_VTEP"] = "1"
+
+		l := len(option.Config.VtepEndpoints)
+		cDefinesMap["VTEP_NUMS"] = strconv.Itoa(l)
+
+		ip_str := "(__u32[]){"
```

Comment:
Drive-by: I think a `strings.Builder` would be more appropriate here. Go re-allocates strings on concatenation. Not a huge problem, but will allow you to get rid of the verbose `x = x + ...` below.